### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by honghuzi** [microsoft/vscode-jupyter#4237]
- Issue Type: `Bug`
- Extension Name: `python`
- Extension Version: `2020.12.422005962`
- OS Version: `Windows_NT x64 10.0.14393`
- VSCode version: `1.52.0`

:warning: We have written the needed data into your clipboard. Please paste! :warning:
```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 270,
		"activateCallTime": 28,
		"activateResolvedTime": 283,
		"activationReason": {
			"startup": false,
			"extensionId": {
				"value": "ms-toolsai.jupyter",
				"_lower": "ms-toolsai.jupyter"
			},
			"activationEvent": "api"
		}
	},
	"runtimeErrors": [
		{
			"name": "Error",
			"message": "request cancelled"
		},
		{
			"name": "Error",
			"message": "request cancelled"
		}
	]
}
```

**Comment by IanMatthewHuff** [comment]
@honghuzi Thanks for reporting and for providing the clipboard info. We'll look into this, but response might be a bit slow until after the new year here.

**Comment by honghuzi** [comment]
@linminhtoo Thank you! rolling back to older versions works for me.

**Comment by honghuzi** [comment]
@IanMatthewHuff This bug can make coding via ssh very very slow. For example, the fast cursor move via MetaGo extension would in this version delay 3~5 seconds during ssh coding. Besides, it takes +10 seconds for jupyter in the python interactive windows to calculate '1+1'. 
I don't know what the bug really is, but the "show running extensions" page showed that only Python extension threws an uncaught exeption.

**Comment by rchiodo** [comment]
There's not a lot of detail here, but remote WSL works fine for me in our latest bits. There's no slowdown whatsoever. 

@honghuzi  can you outline your setup and what exactly the problem is? It sounds like you're remoting to linux machine and your connection is slow? If so I don't believe that would be the jupyter extension causing that problem.

What did you rollback? VS code or the python extension?

**Comment by honghuzi** [comment]
This bug seemed to be resolved in the latest version, 1.53.0

**Comment by ntek-pengsoo** [comment]
@honghuzi Thanks v1.53.0 works for me
